build-tasks/common/meta.js

```diff
@@ -2,11 +2,24 @@ const path = require('path');
 const glob = require('glob');
 const { readJSON } = require('./files');
 
+/**
+ * @see https://github.com/Microsoft/tslint-microsoft-contrib/issues/694
+ */
+const ignoredRulesNotYetAdded = ['banTsIgnoreRule.js', 'commentTypeRule.js', 'noDefaultImportRule.js', 'unnecessaryConstructorRule.js'];
```

Comment:
With exclusion list in this function - new TSLint rules will also be excluded for `rule-metadata.json` and `tslint-warnings.csv` (this change is missing from this PR), however as I understand - main reason is to avoid changes in `recommended_ruleset.js` (and then package `tslint.json`).

Therefore I think it will be better to list of excluded rules to `generate-recommendations.js` or separate file (if more minor updated are possible before `7.0`). This will exclude rules from recommendation, but `tslint-warnings.csv` will still list them as recommended (or not if specified in `additional_rule_metadata.json`). It will also allow checks by rule name instead of file names.